pom.xml

```diff
@@ -413,6 +414,11 @@
                 <version>${mapdb.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.threeten</groupId>
+                <artifactId>threeten-extra</artifactId>
+                <version>${threeten.version}</version>
+            </dependency>
```

Comment:
This dependency is not well sorted: in fact the two previous dependencies too :/